core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/RawlsInstrumented.scala

```diff
@@ -69,6 +71,7 @@ trait RawlsInstrumented extends WorkbenchInstrumented {
     */
   protected def opportunisticEntityCacheSaveCounter: Counter =
     ExpandedMetricBuilder
+      .expand(StatsBoardMetricKey, RawlsBoardMetricValue)
       .expand(WorkspaceMetricKey, "opportunistic_entity_cache_save")
```

Comment:
My understanding was that we would replace "workspace/opportunistic_entity_cache_save" with "<new metric key>/opportunistic_entity_cache_save". Looks like we haven't replaced it here?